---
<!-- title: Dissertation Outline -->
<!-- author: Michael Chambers -->
bibliography: papers2_bibtex.bib
csl: styles/cell.csl
---

## Dissertation Title

### List of Figures
### List of Tables
### Acknowledgments
### Vita
### Selected Publications
### Abstract of the Dissertation

### Chapter I: Introduction: Groucho and _Drosophila_ development

* Sections:
	* The Groucho/TLE family of corepressors are ubiquitous regulators of animal development
	* The domain architecture of Groucho/TLE family proteins
	* Groucho integrates multiple signaling pathways to generate specific cellular responses and fates 
	* Post-translational modifications of Groucho (WIP)
	* Groucho is an essential component of the embryonic axial patterning network
	* Groucho is capable of both short- and long-range repression
	* The mechanism of Groucho-mediated repression

* Groucho/TLE family: structure and conservation
	* Member of one of a prevalent group of corepressors conserved throughout animals
	* Discovery
		* Weak hypomorphic allele resulting in extra supraorbital bristles {Lindsley, 1968 #3055}
		* Stronger mutations are lethal {Paroush, 1994 #172}
	* Prevalent in higher eukaryotes
		* Model organisms
			* Xenopus {Choudhury, 1997 #3121}
			* Mouse {Mallo, 1993 #3123}
			* Nematode {Pflugrad, 1997 #3125}
			* Rat {Schmidt, 1993 #3126}
			* Zebrafish {Wulbeck, 1997 #3127}
		* Human {Miyasaka, 1993 #3124}
		* One copy in fly and nematode, multiple in higher organisms {Li, 2000 #3128}
			* Four members TLE1-4
			* Four isoforms in human {Stifani, 1992 #3065}
				* Non-redundant roles in development {Yao, 1998 #3069}
			* Two truncated forms in mammals
				* AES {Gasperowicz, 2005 #3129}
					* Disrupt oligomer {Brantjes, 2001 #3058}
					* Preferentially interact with repressors {Brantjes, 2001 #3058} {Muhr, 2001 #3132}
				* Tle6 {Marcal, 2005 #3131}
	* Domain structure {Turki-Judeh, 2012 #2385}
		* N-terminal Q (glutamine-rich) domain highly conserved
			* Responsible for self-association {Chen, 1998 #267} and interaction with some repressors {Chen, 2000 #83}, including the TCF/LEF family {Brantjes, 2001 #3058}
				* Tetramerization not requisite for recruitment to chromatin in vitro, but necessary for production of aggregates {Sekiya, 2007 #1658} 
			* Structure of TLE Q-domain recently solved {Chodaparambil, 2014 #3057}
				* Forms a dimer of dimers
			* Does not effect recruitment pattern in S2 cells {Kaul, 2014 #2204}
			* Not required for all Gro-mediated repression {Jennings, 2007 #2990}, potentially due to differences in mechanism or interactions
		* C-terminal WD domain
			* Highly conserved
			* Human homologue structure solved, winged propeller domain it
			* Binds to repressors through interactions with peptide motifs binding in central cavity {Jennings, 2006 #3059}
			* Mouse homologue, also necessary for chromatin association {Sekiya, 2007 #1658}
		* Central domains
			* Protein interactions and Groucho regulation regions
			* Three domains GP, CcN, and SP
			* GP
				* Binds to HDAC1/Rpd3 {Chen, 1999 #3061}
				* Also potentially nuclear localization sequence {Turki-Judeh, 2012 #2966}
			* CcN
				* CK2 and Cdc2 phosphorylation sites, regulated by cell cycle {Nuthall, 2002 #3062}
			* SP
				* Multiple phosphorylation sites (MAPK) {Hasson, 2005 #3064}
				* Down-regulation of Gro following EGFR signaling, with this effect lasting after EGFR signaling ceases {Helman, 2011 #2938} {Hasson, 2005 #3064}
			* Evidence that Gro central domains are intrinsically disorderd {Turki-Judeh, 2012 #2966}
				* Also play a role in target specificity or constriction of Groucho repressive activity
* Developmental roles of Gro
	* Outside of Drosophila, many roles, overall poorly understood
	* Single isoform in drosophila
		* Humans {Stifani, 1992 #3065} and mice {Mallo, 1993 #3066} each express four non-redundant Gro homologs (TLE1-4 and Grg1-4, respectively) {Yao, 1998 #3069}
		* Humans additionally express a truncated TLE, termed AES, which associates with full length TLE to downregulate TLE-mediated response to Wnt signaling {Itatani, 2015 #123}
		* In humans, Gro/TLE is involved in numerous developmental processes, including organ development, neurogenesis, hematopoesis, and osteogenesis {Bajoghli, 2005 #3068} {Javed, 2000 #3070}
	* In drosophila
		* Diverse roles, many well-studied
		* Integration of multiple signaling pathways and interpretation of morphogen gradients
			* Ras/MAPK {Cinnamon, 2008 #242}
				* 
			* Notch {Barolo, 2002 #3072}
				* Canonical notch signaling involves a relatively simple cascade of events (at least biochemically)
					* Notch receptors interact with one of a number of intracelullar activating ligands
					* Results in cleavage of the receptor, releasing Notch^ICD (Notch intercellular domain)
					* Rapidly diffuses to nucleus
					* Staggered gene activation through feed-forward loop (initially activated E(spl) genes contribute to additional, dramatic changes in gene expression)
				* Under absence of signaling, Gro represses E(spl) complex genes through interaction w/ Hairy {Delidakis, 1991 #3082}
				* Upon signaling, E(spl) genes become activated (Notch displaces Gro binding to Su(H)) and Gro recruited to E(spl) to repress proneural genes {Preiss, 1988 #3083} {Wurmbach, 1999 #3084}
			* decapentapletic (dpp)/TGF-beta homologue {Upadhyai, 2013 #2339}
				* Multiple roles in appendage patterning via activity within imaginal disks later in development
				* Morphogen expressed dorsally in embryo; crucial for specifying dorsal cellular fates {Ferguson, 1992 #3088}
					* Expression mediated by Drosal/Groucho
				* In the absence of dpp signaling, Brinker (brk) represses a subset of dpp target genes via interaction w/ Gro and dCtBP {Hasson, 2001 #3033}
				* Upon signaling activation, brk becomes repressed by Schnurri in dorsal regions of the embryo; brk continues to be expressed ventrolaterally {Marty, 2000 #3089}
			* Wingless/Wnt {Cavallo, 1998 #3071} {Roose, 1999 #3086}
				* Tcf/Lef target genes repressed via Gro interaction in unstimulated cells {Clevers, 2006 #3085}
				* Beta-catenin nuclear concentration increases via signaling, displacing Gro and activating target genes {Daniels, 2005 #3087}
			* Groucho is a critical component of a diverse array of signaling pathways
				* Hundreds of cell types in adult fly, yet far fewer signaling pathways {Perrimon, 2012 #3175}
				* Informational content of signal not binary but is always interpreted in context of additional pathways
					* Necessitates that the logic of such a system is non-additive
					* Communal utility of and competition for repressive elements is one way to address this need
				* Allowing the integration of multiple inputs to determine a much larger number of outputs, these outputs then determining a smaller set of cellular fates
				* Control on multiple levels
					* Availability of interacting partners
						* Partner availability, distribution, and affinity
					* Repressive power through PTM
				* Not just a passive component of repression
				* By understanding Groucho's role in these pathways, we gain insight into the pathways themselves
		* Post-translational modifications of Groucho
			* Extensively PTM'd, primarily in central, intrinsically disordered domain {Turki-Judeh, 2012 #2385}
			* Phosphorylation
				* (derepression) MAPK signaling, as mentioned in the previous section, to down-reg Gro activity
					* Phosphorylated in termini and seven transverse stripes in early (cellular blastoderm) embryos
					* Terminal patterning Torso {Helman, 2012 #2991}
					* Expression of genes in the ventral neuroectoderm EGFR (ind) {Cinnamon, 2008 #3077}
					* Phosphorylation of Gro inactivates repression, but does not result in nuclear export or degradation {Cinnamon, 2008 #3077}
				* (inc repression) 
		* Dorsal-ventral patterning
			* Maternally-contributed gradient of nuclear dorsal {Roth, 1989 #1112}
			* On ventral side of embryo, nuclear dorsal initiates cellular programs designating the mesoderm {Gonzalez-Crespo, 1993 #3043}
			* On ventrolateral regions, modest dorsal concentration leads to neuroectodermal fate {Ip, 1992 #3042}
			* Strength, patterning of dorsal binding sites, and adjacent binding sites determines dorsal activity {Zeitlinger, 2007 #3025}
		* Anterior-posterior patterning
			* Terminal patterning via interaction with capicua {Ajuria, 2011 #2947}
				* In response to Torso RTK signaling {Chen, 2009 #3073} {Paroush, 1997 #3074}
			* Stripe definition and polarity via engrailed interaction {Jimenez, 1997 #3075} {Kobayashi, 2001 #3076}
				* Specification of strips (dmel embryo split into 14 stripes which specify the segments of the adult body)
		* Imaginal discs
			* Wing patterning in wing discs (dpp) {Winkler, 2010 #2964}
* Mechanism of repression
	* Oligomerization/spreading/looping
		* Follows from:
			* Groucho tetramerizes; possible evidence for higher-order structures {Chen, 1998 #267} {Song, 2004 #1161}
			* Groucho occupancy detected far from recruitment site by ChIP {Martinez, 2008 #2287}
				* Analogous to Sir family of corepressors {Pirrotta, 2005 #3106}
			* Evidence from mouse homolog {Sekiya, 2007 #1658}
				* Tetramerization not required for recruitment to chromatin, or increase in nucleosome density
					* Or localized spreading ( ~ 500 bp)
				* Tetramerization is required for aggregation of separate nucleosome arrays
				* Blocks recruitment to sites, but difficult to fit with picture of Gro as short- and long-range repressor
		* Arguments agains:
			* Required for viability, but not necessary for all repressive activity in vivo {Jennings, 2007 #2990}
			* Evidence in cell culture showing discrete peaks, few wide peaks {Kaul, 2014 #2204}
	* Recruitment of histone deacetylases
		* Preferentially associates with deacetylated histone tails {Flores-Saaib, 2000 #656}
		* And Rpd3 {Chen, 1999 #3105}
		* Some repressive activity dep on Rpd3, which colocalizes w/ Gro and increases nucleosome density {Winkler, 2010 #2964}
		* Fails to completely explain repressive ability
	* Competition w/ activating factors
		* Pax2 etc
	* DNA topological considerations
		* ...
* Groucho and disease


### Chapter II: Groucho activity in the developing embryo
#### Abstract
#### Introduction

* General Gro overview
	* Transcriptional corepressor
	* Involved in multiple developmental patterning systems, including axial and terminal patterning
	* Project overview
		* Deep-sequencing technologies to probe Groucho localization, targets, and obtain functional insights
* Groucho and development
	* Transcriptional corepressor
	* Spatial and temporal restriction of gene expression
	* Interacts with numerous sequence-specific transcription factors
		* Generally through interactions with conserved peptide motifs
	* Through these interactions, regulated tissue specification at multiple steps in development
	* Groucho as a decision node, integrating signal information combinatorially and in a context-dependent fashion, as it's activity is dependent on the presence or absence of one or more interacting partners
	* Groucho is conserved throughout metazoans
		* Implicated in developmental abnormalities and disease in humans
* Despite centrality, no attempt to explore Groucho's activity in the developing embryo
	* To this goal, established multiomics approach in wild-type and Groucho mutant embryos	
	* Roles throughout Drosophila development
	* Ubiquitously expressed
	* Roles in tissue specification in response to multiple extracellular signaling pathways (RTK signaling being most prominent)
	* Significant points in timepoints under analysis
* RNA-seq for transcriptome analysis
	* Allows for the analysis of Groucho regulatory activity in a developmental context
	* Development is a rapid and fluid process, so necessary and instructive to analyze groucho activity at multiple timepoints throughout the process
	* Interested in looking at Groucho activity at the initiation of zygotic transcription onwards, though Grouch is maternally deposited and so ubiquitously present in the embryo before this time
* Short reiteration of major developmental events w/ Gro involvement at each of the three timepoints analyzed

#### Materials & Methods

* Fly strains/culturing/collection
	* All fly lines were grown according to established procedures (look up how exactly to cite this)
* RNA isolation/library generation/sequencing
	* RNA isolated with Qiagen RNA extraction kit (look up name)
	* RNA quality assessed by Agilent Bioanalyzer
	* Stranded RNA-seq libraries generated using Illumina TruSeq transcriptome generation kits (look up name of kit I suppose, or ask Sean)
		* Libraries were barcoded for multiplex sequencing
	* Sequencing performed at UCLA High-Throughput sequencing core (name) on ____ (machine? Illumina HiSeq something probably) 
* Transcriptome (RNA-seq) data preparation and genomic alignment
	* QC/filtering
		* Quality assessment performed with FastQC (no available publication)
	* Demultiplexing performed by in-house scripts (github link?)
	* Mapping
		* Mapping performed using Tophat2 v2.0.9 {Kim, 2013, r08998}
		* Using Drosophila assembly BDGP?/dm3 (Illumina iGenomes)
		* Transcript accessions (dm3) provided by FlyBase (Illumina iGenomes)
	* Assignment of mapped fragments to genes performed by HTSeq
* Bioinformatics
	* Differential expression analysis w/ DESeq2 {Love, 2014 #3031}
	* SKIP: Differential expression analysis w/ MaSigPro {Nueda, 2014 #3032}
	* 

#### Results

* Groucho is recruited ubiquitously and dynamically throughout the genome
	* Major events in development occuring during the three time windows
	* Sequencing overview
		* Table: Total reads/qc pass reads/mapped reads/reads by feature (intergenic/intronic/exon/etc)
		* Replicate similarity
			* Heatmap of spearman coefficients
			* PCA analysis
			* Characteristic MA plot
	* Comparisons to modENCODE consortium data
		* Overlap with 0 - 12 hour embryo data
		* Overlap with white pre-pupae data
			* Indicative of some common regulatory roles throughout all (potentially) stages of development
	* Gro recruitment dynamic and reversible
		* 50% of sites unique to single timepoint
* Groucho associates with multiple ventrally-repressed genes in patterns unique to early embryonic stages
	* zen
	* dpp
	* tld
* Groucho localizes extensively to the Dorsal-binding sites of both Dorsal-activated and â€“repressed genes
	* Reliance on additional factors to facilitate Dorsal binding
	* snail
	* twist
	* rho
	* simple minded
	* short gastrulation
	* Groucho and Dorsal sites
		* Classes of Dorsal site
		* Alternative targets of Dorsal repression (segmentation and homeotic patterning)
			* Groucho known to regulate these genes
				* Engrailed and Hairy family proteins
* Groucho-associated genes tend to be associated with multiple, small, clustered peaks
	* RENAME: Groucho tends to bind in spatially-restricted clusters at promoters and inside genes
* Groucho binding is enriched in promoters, UTRs, and introns
* Groucho colocalizes with Dorsal to areas of high transcription factor density in the very early embryo
* Identification of Groucho Targets by Developmental Stage
	* High-confidence target identification
	* Scoring algorithm integrating ChIP-seq and RNA-seq data
	* Observations
		* LoF data contains approximately equal numbers of activated and repressed genes
		* OE data strongly enriched for repressed genes
		* Asymmetric response -> potentially indicative of primary role as repressor
		* 

* Previous outline fragments
	* Groucho recruitment is more dynamic during early embryonic stages (consistent with highest levels of Gro at these stages)
	* Groucho spreading does not appear to be an obligate characteristic of repression
		* Groucho peaks exhibit a mean width of approximately 500 bp
		* Groucho widths slightly increase in size at later timepoints, though whether this is the result of biological variability or slight deviations in chip-seq library fraction efficiencies is uncertain. In either case, the effect is slight.
		* Clustering of Gro binding regions does appear to be prevalent. Over half of all Groucho-associated genes exhibit two or more distinct Groucho binding sites
	* Groucho binds at transcription start sites and introns

	* At all timepoints, about half of Gro binding is either upstream or overlapping the start site of the nearest gene feature, consistent with Groucho's association with enhancer regions, which are typically, but certainly not always upstream of their target gene
		* Approximately 40% of all Gro binding sites are contained with gene bodies, with 35 to 50% of those sites falling within an intron. Of those, 40% are found within the first intron.
		* In Drosophila, the first intron is typically longer, and most relevant to gene regulation.
		* It is unknown whether Groucho is being actively recruited to these intronic regions, or if looping from an enhancer region is responsible for depositing Gro within these areas.
		* Motif analysis of binding sequences within introns reveals few conserved sequences, possibly indicating the lack of a sequence-specific DNA binding factor being responsible for recruiting Gro at many of these positions
			* However, recruitment of Gro to introns seems highly specific (is being directed by some factor or process), as it often occurs with 100-200 bps of the TSS, in discreet peaks or clusters of peaks
			* Several examples of genes with alternative TSS's exhibiting Groucho peaks in introns adjacent to each start site (ex. dpp)
			* Some identified motifs:
				* TP1
					* twist (21 sites)
	* Perturbation of Groucho levels results in the misregulation of thousands of genes
		* Wild-type and mutant expression profile of Groucho shows ubiquitious maternal contribution in the early embryo, and steep fall-off at later timepoints
			* Overexpression shows higher expression at early timepoints, and a slight recovery towards wt levels aftewards
			* Gro LoF shows constitutively low levels of Groucho expression
			* Is Gro zygotically expressed at all?
			* According to modENCODE data, gro transcript levels are highest at embryo deposition, and steadily decrease, reaching a low point during larval L3 stage. Gro levels are slightly elevated during late L3 and white prepupae stages, before dropping once more to approximately early/mid larval levels
			* Our Gro profile is consistent with these observations	
			* Figure: Groucho expression timecourse (wt and mut)
	* Groucho target genes are enriched for developmentally-regulated transcription factors
	* Drosophila Gro is a dedicated repressor
		* Introduce concept of Gro homologs that serve primarily as activators
		* Evidence from LoF and OE regulated genes

	* Recruitment of Groucho to intronic regions correlates with repression of those genes, but does not appear to be an obligate feature of repression
		* Multiple known or suspected Gro targets lack introns (zen, tolloid, twist, wntD, tll, sna)
	* Some genomic areas exhibit widespread Groucho binding, especially around Gro targets in areas with low gene density
		* Ex. snail; 21 distinct Groucho bindng sites spread over a 50kb region
		* Ex. odd skipped; 17 distinct sites over 30kb
	* Groucho involved in active, reversible repression is enriched for upstream and within genes
	* Groucho regulated genes are enriched for paused PolII, and depleted for actively transcribing PolII
		* Stronger enrichment with genes with Gro intronic binding
	* Groucho localizes in distinctive patterns adjacent to, and overlapping, predicted target genes
		* Capicua-regulated genes
		* Brinker-regulated genes
		* Zelda interaction

	* Data overview
		* Sequencing depth/replicate similarity
			* Start with Gro LoF and introduce overexpression lines later?
	* Groucho loss-of-function data
		* Number of differentially expressed genes per dataset (**Table**)
		* Up and down-regulated genes per timepoint (**Table**)
			* Overlap of DE genes between timepoints (Figure: venn diagram)
	* Groucho overexpression data
		* Plot of Groucho mRNA levels by timepoint
		* Some evidence of a dosage response to Groucho overexpression when comparing the 2-fold and 4-fold data sets, though the majority of differentially expressed genes exhibit near-identical levels of repression
			* Possible interpretations
				* Groucho functions primarily as a binary regulator
				* The limiting factor in dosage response is the concentration of the associated repressor instead of Groucho
		* Overexpression of the deltaSP mutant results in similar changes in expression to the wild-type overexpression lines
			* This is based on correlation data and looking at some individual genes; not sure how to best visualize this
			* This confirms that this domain is not essential for Groucho activity
			* Does not necessarily support the hypothesis that this domain is responsible for down-regulation of Groucho activity or in the definition of Groucho targeting
			* There may be more subtle differences between the deltaSP and wild-type overexpression datasets (worth looking into)
	* Similarity/inverse similarity of Groucho LoF and OE lines
		* **Figure:** venn diagram
		* Group analyses of genes showing opposite responses to Gro LoF vs OE (GO analysis, patterns in the chip-seq data)
		

#### Discussion

* Gro recruited dynamically and transiently
* Dorsal targets
	* Recruitment within and outside regulatory regions
	* Multiple peaks per gene
	* 


* Dynamics of Developmental Groucho Genome Occupancy
	* Large number of developmentally specific binding sites
		* Widespread rearrangement of occupancy following early stages
			* Indicative of specific roles for Gro in early gene regulation, that are not carried over
	* Large number of sites that are occupied throughout development (should motif scan these?)
	* Groucho regulation is transient and generally non-repeatable (implies high cell-type temporal specificity)
	* Association of Groucho with Dorsal-target genes
	* Binds within known enhancer regions of zen and dpp
	* Additionally overlaps transcription start site
	* Potential direct or mediated interaction with transcriptional machinery
* Preference of sites for binding within genes
	* Significant preference for introns
	* Significant preference for initial introns
	* Consistent with observations of other Dmel TFs
		* kruppel {Matyash, 2004 #3046}
		* twist {Sandmann, 2007 #3048} {Zeitlinger, 2007 #3025}
		* wei {Wei, 2006 #3047}
		* First introns are longer and more conserved {Bradnam, 2008 #3034}
	* Intron binding is common motif of activators and repressors
	* Hypothesize as to why
		* Interactions w/ polymerase at promotor only block initation of novel transcription
		* As drosophila polymerase traverses ~ 1 kb per min, introduces significant lag in complete repression
		* Post-transcriptional regulation common method of curtailing run-on trancription
		* Perhaps repressor binding additionally assists to stop elongating polymerases (would be helpful to have a metagene here)
* Breadth of peaks suggests point-source association
	* Implications for Groucho mechanism
	* Enrichment for genes with paused PolII 

* Enrichment of Gro targets for TFs => early regulatory factor in cascading transcriptional changes
* Enrichment for paused PolII
	* Developmentally regulated genes often have paused PolII



### Chapter III: Nascent-seq
#### Abstract
#### Introduction

* Drosophila development and transcriptome {Brown, 2014 #3144}
	* Highly dynamic
	* Tightly controlled spatially and temporally
	* ~7000 distinct transcripts present from the onset of development {Graveley, 2011 #3044}
		* Nearly ~12,000 in adults males / ~9000 in adult females 
			* Due to testis complexity {Parisi, 2004 #3145}
		* Arising from ~3000 genes, ~1000 of which are spatially restricted {De Renzis, 2007 #3149} {Tomancak, 2002 #1150}
	* Embryos go to great lengths to limit stochastic fluctuations in expression level {Perry, 2010 #385}
	* Ideally measure protein levels {Fu, 2009 #3150}
* Additional complexity arising from maternally contributed genes
	* Prevalent in animals {Tadros, 2009 #3146}
	* In flies, two stages {Pritchard, 1996 #3147}
		* Low levels very early transcription
			* Small number of genes, including Engrailed, a gro interactor {Ali-Murthy, 2013 #3148}
		* "Minor wave" ~ 1.5 hr
		* "Major wave" ~ 2.5 hr
* Investigate the dynamics of transcript expression
	* mRNA levels as measured a function of multiple processes
		* Transcription rate/level (accumulation)
		* Processing
		* Degradation {Thomsen, 2010 #1350}
* Introduction to nascent-seq
	* Transcription rate determination
		* Importance to non-steady state transcript level analysis
	* Fractionation methodology

#### Materials & Methods

* Flies
	* Strains
	* Culturing
	* Collection
* Isolation of chromatin-associated RNA
	* Embryo fractionation
	* RNA isolation
	* Validation of fractionation (anti-H3/anti-tubulin immunoblot)
	* Depletion of rRNA
		* Sequences of depletion oligomers
	* Validation of mRNA quality (Bioanalyzer)
* Library preparation/QC
* Sequencing
* Data processing
	* QC/pre-processing
	* Mapping
	* Transcript assignment and expression level normalization

#### Results

* 


* Validation of chromatin-associated RNA isolation
	* anti-H3/anti-tubulin immunoblot
	* Bioanalyzer profiles/pseudogels
	* Schematic of rRNA depletion procedure(?)
* Overview of data sets
	* Reads per sample
	* Intra- and inter-replicate similarity analyses
* Primary differences with total mRNA seq experiments

#### Discussion

* 

#### Figures & Tables
#### References

### Appendices
#### Appendix X: Groucho-regulated gene data
#### Appendix X: Groucho binding regions by timepoint
