1)  Groucho targets evolve as a function of developmental stage.

    a.  Combine the RNA-seq (primarily relying on the loss of function
        data) and ChIP-seq analysis to come up with a set of high
        confidence direct Groucho targets

    b.  Compare to overexpression data. To what extent does the
        overexpression data confirm the loss of function data? Not sure
        if we want to include the ∆SP data or not. If it is just the
        same as the wt overexpression data, then I’m not sure I see the
        point in including it.

    c.  Address the issue of whether Gro is likely to be exclusively a
        repressor or also an activator

    d.  Some kind of clustering analysis to group genes according to
        which stage they are regulated

2)  Nascent-seq analysis reveals unexpected differences with analysis of
    steady state mRNA levels. Questions to be addressed:

    a.  Do Groucho target genes reveal different patterns (for example,
        in principal component analysis) with regard to the
        RNA-seq/nascent-seq comparison than the transcriptome as a
        whole?

    b.  How do these patterns vary depending on developmental stage?

    c.  After addressing a and b, can we draw conclusions about what
        underlying similarities and differences are responsible for the
        similarities and differences observed in the principal component
        analysis?

3)  Analysis of Gro binding sites suggests mechanisms for Gro-mediated
    repression.

    a.  How are Groucho binding sites distributed with regard to TSS? Is
        this different for genes that (according to the RNA-seq
        analysis) are repressed by Groucho, genes that are activated by
        Groucho, and genes that don’t appear to be regulated by Groucho?
        Does it differ according to developmental stage?

    b.  Does Gro remain associated with genes even at time points when
        it is not regulating those genes?

    c.  What meaningful comparisons can we make between Groucho binding
        sites and patterns of histone modification as revealed by
        publically available histone modification data, data regarding
        nucleosome positioning and nucleosome-free regions, and binding
        sites for sequence-specific transcription factors (especially
        those, like Dorsal, known to interact with Groucho)? I know that
        the timepoints won’t be the same, but we can still try to make a
        comparison.

    d.  What motifs are enriched in Groucho binding sites? Do we see
        sequence signatures associated with specific families of genes
        (repressed, activated, not regulated, regulated with a certain
        temporal profile, etc.)?


